nest/03.nest.md

```diff
@@ -0,0 +1,89 @@
+# 2 nest
+In this chapter, we will detail how to use the binary `nest`.
+As explained before, this binary is mainly designed for ease of use.
+However, anyone can use it.
+
+## 2.1 nest options
+`nest` has three options available:
+
+* `-h` or `--help`, which prints the help information of `nest`, consisting of a short explanation for each of `nest`'s options and commands.
+* `-v`, which sets the level of verbosity for `nest`. Basically, how much `nest` will explain what it's doing in the terminal.
+* `-V` or `--version`, which prints the version of `nest` you are currently using.
+
+To use them, simply type `nest` followed by any option. For example, `nest -h`.
+
+## 2.2 nest commands
+### 2.2.1 Updating the list of available packages with `nest pull`
+This command synchronizes the cache of local packages with the one on Raven's server.
+In other words, `nest pull` updates the list of available packages for you to install with Nest.
+You have to run this command frequently (at least before installing or upgrading a package) in order to have an up to date cache.
+Otherwise, you might not be able to install new packages, or new versions of packages that were uploaded after the last time you ran `nest pull`.
+
+#### Common issues
```

Comment:
Instead of having a sperate `Common issues` section, maybe use the infobox we discussed (Unless the `Common issues` section is long, but that's not the case here)?

What do you think of that?

Reply:
good idea, I totally forgot about it this since the last meeting we've had D:

Reply:
I was thinking of using an infobox just before the Common Issues section to say how important it is that the local cache is kept up-to-date. But by doing that, there would be two infoboxes in a row (that one, and the common issues one), which might look a bit weird. What do you think about this?

Reply:
I do not think it's weird to have two infobox in a row, especially if they are of different kind (and therefore will have different colors)